pkg/ansible/paramconv/paramconv.go

```diff
@@ -70,6 +71,41 @@ func ToCamel(s string) string {
 	return ret
 }
 
+//This function modifies the string to handle special words that are a part of wordMapping
+//Add values to handle wider number of cases
+func preprocessWordMapping(s string) string {
+	count := 0
+	length := len(s)
+	for i, v := range s {
+		if i+1 < length {
+			next := s[i+1]
+			count++
+			if (v >= 'A' && v <= 'Z' && next >= 'a' && next <= 'z') || (v >= 'a' && v <= 'z' && next >= 'A' && next <= 'Z') {
```

Comment:
I don't like the deep indentation. What if we flip the logic of the if. So let's do this instead:

```
for i, v := range s {
    if i+1 > length {
        continue
    }

    next := s[i+1]
    count++
    ...
}
```

Doing this the code won't have to be nested so deep for the one edge case where `i+1 > length`.